processor/deltatocumulativeprocessor/internal/data/expo/scale_test.go

```diff
@@ -45,6 +45,18 @@ func TestDownscale(t *testing.T) {
 	}, {
 		{scale: 2, bkt: "1 1 1 1 1 1 1 1 1 1 1 1"},
 		{scale: 0, bkt: "   4       4       4   "},
+	}, {
+		{scale: 1, bkt: "ø 1 1 0"},
+		{scale: 0, bkt: " 1   1"},
+	}, {
+		{scale: 1, bkt: "ø 1 1"},
+		{scale: 0, bkt: " 1   1"},
+	}, {
+		{scale: 1, bkt: "-  1 1"},
+		{scale: 0, bkt: "- 1   1"},
+	}, {
+		{scale: 5, bkt: "- 5 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 9 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 3 0 0 0 0 0 0 0 0 0 0 0 0 5 0 0 0 0 0 0 0 0 0 0 5 0 0 0 0 0 0 0 3 0 0 0 0 0 0 8 0 0 0 0 0 6 0 0 0 0 0 6 0 0 0 0 5 0 0 0 5 0 0 0 2 0 0 0 5 0 0 7 0 0 0 4 0 5 0 0 4 0 0 3 0 3 0 0 8"},
```

Comment:
The previous ones make sense to me, but I have to admit that I have no idea about this and the next test case. I'm trusting the folks reviewing this before me and approving this PR.

Reply:
agreed, this while I _think_ its correct this case is overly complex and as such of limited use. can we come up with a simpler one instead?

Reply:
done